lib/galaxy/authnz/custos_authnz.py

```diff
@@ -187,7 +187,10 @@ def _load_config_for_provider_and_realm(self, provider, realm):
     def _get_well_known_uri_for_provider_and_realm(self, provider, realm):
         # TODO: Look up this URL from a Python library
         if provider == 'custos':
-            return "{}/realms/{}/.well-known/openid-configuration".format(self.config["url"], realm)
+            base_url = self.config["url"]
+            # Remove potential trailing slash to avoid "//realms"
+            base_url = base_url if base_url[-1] != "/" else base_url[:-1]
```

Comment:
Maybe it would be better to fix this in the manager so that it would be applied on other backends in addition to custos.